cognite/client/_api/simulators/simulator_models.py

```diff
@@ -160,3 +165,69 @@ def retrieve_revision(self, id: int | None = None, external_id: str | None = Non
             resource_path="/simulators/models/revisions",
             headers={"cdf-version": "beta"},
         )
+
+    @overload
+    def create_model(self, model: Sequence[SimulatorModel]) -> SimulatorModelList: ...
+
+    @overload
+    def create_model(self, model: SimulatorModel | SimulatorModelWrite) -> SimulatorModelList: ...
+
+    def create_model(
+        self, model: SimulatorModel | SimulatorModelWrite | Sequence[SimulatorModel] | Sequence[SimulatorModelWrite]
+    ) -> SimulatorModel | SimulatorModelList:
+        """`Create one or more simulator models. <https://api-docs.cognite.com/20230101-beta/tag/Simulator-Models>`_
+
+        You can create an arbitrary number of simulator models, and the SDK will split the request into multiple requests.
```

Comment:
Ahh, ok, so will the SDK iterate over the list of models and create multiple requests then?

Reply:
If we support multiple models, we should rename the input argument to `models` instead of `model`